release-please-config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "packages": {
+    ".": {
+      "release-type": "python",
+      "package-name": "cognite-client",
+      "bump-files": [
+        "pyproject.toml",
+        {
+          "filename": "cognite/client/_version.py",
+          "search-regex": "__version__ = \"(.*)\""
+        }
+      ]
+    }
+  },
+  "changelog-types": [
+    { "type": "feat", "section": "✨ Features", "hidden": false },
+    { "type": "fix", "section": "🐛 Bug Fixes", "hidden": false },
+    { "type": "perf", "section": "⚡ Performance Improvements", "hidden": false },
+    { "type": "docs", "section": "📚 Documentation", "hidden": true },
+    { "type": "style", "section": "🎨 Styles", "hidden": true },
+    { "type": "refactor", "section": "🛠 Refactoring", "hidden": true },
+    { "type": "test", "section": "✅ Tests", "hidden": true },
+    { "type": "chore", "section": "🔧 Chores", "hidden": true }
+  ]
+}
```

Comment:
Will this make changelogs in the same format as we currently have? The current changelog has the following groupings, which does not quite match this:
```md
Changes are grouped as follows
- `Added` for new features.
- `Changed` for changes in existing functionality.
- `Deprecated` for soon-to-be removed features.
- `Improved` for transparent changes, e.g. better performance.
- `Removed` for now removed features.
- `Fixed` for any bug fixes.
- `Security` in case of vulnerabilities.
```